.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+
+language: go
+go:
+- 1.5
```

Comment:
Both? Or just 1.6.2? I simply used the node-exporter as example. I'll change it for now to 1.6.2.